pages/settings/teams.tsx

```diff
@@ -1,46 +1,57 @@
 import Head from 'next/head';
-import prisma from '../../lib/prisma';
-import Modal from '../../components/Modal';
 import Shell from '../../components/Shell';
 import SettingsShell from '../../components/Settings';
 import { useEffect, useState } from 'react';
-import { useSession, getSession } from 'next-auth/client';
+import { useSession } from 'next-auth/client';
```

Comment:
Codacy has a fix for the issue: [Replace `'next-auth/client'` with `"next-auth/client"`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689300)
```suggestion
import { useSession } from "next-auth/client";
```